graph/collection-properties.md

```diff
@@ -0,0 +1,402 @@
+
+There is no way to do this for complex types because there is no way to address a particular instance of a complex type within a collection. 
+
+### Entity Type
+
+```HTTP
+GET  /interestingData/bars/firstBarId
+
+200 OK
+{
+  "id": "firstBarId",
+  "differentProperty": 10
+}
+```
+
+## Removing individual elements from a collection
+
+The [OData standard](https://docs.oasis-open.org/odata/odata/v4.01/odata-v4.01-part1-protocol.html#sec_DeleteanEntity) specifies that clients can remove individual elements of a collection of entity types using a `DELETE` request:
+> To delete an individual entity, the client makes a DELETE request to a URL that identifies the entity.
+
+There is no way to do this for complex types because there is no way to address a particular instance of a complex type within a collection. 
+
+### Entity Type
+
+#### Remove an element from the collection
+
+```HTTP
+DELETE  /interestingData/bars/thirdBarId
+
+204 No Content
+```
+
+#### Check the new contents of the collection
+
+```HTTP
+GET  /interestingData/bars
+
+200 OK
+{
+  "value": [
+    {
+      "id": "firstBarId",
+      "differentProperty": 10
+    },
+    {
+      "id": "secondBarId",
+      "differentProperty": -6914
+    }
+  ]
+}
+```
+
+## Updating individual elements in a collection
+
+The [OData standard](https://docs.oasis-open.org/odata/odata/v4.01/odata-v4.01-part1-protocol.html#sec_UpdateanEntity) specifies that clients can update individual elements of a collection of entity types using a `PATCH` request:
+> To update an individual entity, the client makes a PATCH or PUT request to a URL that identifies the entity.
+
+There is no way to do this for complex types because there is no way to address a particular instance of a complex type within a collection. 
+
+### Entity Type
+
+#### Update an element in the collection
+
+```HTTP
+PATCH  /interestingData/bars/firstBarId
+{
+  "differentProperty": "15"
+}
+
+204 No Content
+```
+
+#### Check the new contents of the collection
+
+```HTTP
+GET  /interestingData/bars
+
+200 OK
+{
+  "value": [
+    {
+      "id": "firstBarId",
+      "differentProperty": 15
+    },
+    {
+      "id": "secondBarId",
+      "differentProperty": -6914
+    }
+  ]
+}
+```
+
+## Updating a collection
+
+The [OData standard](https://docs.oasis-open.org/odata/odata/v4.01/odata-v4.01-part1-protocol.html#sec_UpdateanEntity) specifies that clients can replace all elements of a collection of complex types or all elements of a collection of entity types using a `PATCH` request:
+> Collection properties...provided in the payload corresponding to updatable properties MUST replace the value of the corresponding property in the entity or complex type.
+
+The standard also specifies that collections of entities can be updated in a relative fashion using the [delta syntax](https://docs.oasis-open.org/odata/odata-json-format/v4.01/odata-json-format-v4.01.html#_Toc38457777) in a `PATCH` request:
```

Comment:
I'm not sure "relative fashion" will resonate as strongly with what the user is trying to do here. maybe something like "The standard allows updating a collection of entities, including adding, removing, or modifying individual existing entities in a single PATCH request. This is not possible with complex types because there is no way to identity them individually within the collection."